pkgs/development/python-modules/wsgiprox/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, six
+, certauth
+}:
+
+buildPythonPackage rec {
+  pname = "wsgiprox";
+  version = "1.5.2";
+
+  src = fetchFromGitHub {
+    owner = "webrecorder";
+    repo = "wsgiprox";
+    rev = "004870a87959e68ff28ff4362e4f0df28ec22030"; # Repo has no git tags
```

Comment:
```suggestion
    # https://github.com/webrecorder/wsgiprox/issues/8
    rev = "004870a87959e68ff28ff4362e4f0df28ec22030";
```

Reply:
Thanks for creating the issue. I updated the comment.